### DISCUSSION TITLE
Connect to selfhosted anonaddy

### DISCUSSION THREAD

**Discussion by praptoumache** [dani-garcia/vaultwarden#2731] [Ideas]
I'd like to connect my selfhosted anonaddy but found that it is not implemented yet.
https://github.com/anonaddy/anonaddy/issues/313

i'd love to see a featur like that!

**Comment by BlackDex** [comment]
That is not something we can provide. This needs to be build-in into the Bitwarden clients.

**Reply by SamuelNitsche** [reply]
@BlackDex FYI: The PR for the Bitwarden clients was merged and will be released in the near future. 
https://github.com/bitwarden/clients/pull/4056
